src/EFCore.MySql/Query/Internal/MySqlStringMethodTranslator.cs

```diff
@@ -166,6 +174,24 @@ public virtual SqlExpression Translate(SqlExpression instance, MethodInfo method
                     .MakeEndsWithExpression(instance, arguments[0], _sqlExpressionFactory.Constant(StringComparison.CurrentCulture));
             }
 
+            if (_matchMethodInfo.Equals(method))
+            {
+                return new MySqlStringComparisonMethodTranslator(_sqlExpressionFactory)
+                    .MakeMatchExpression(instance, arguments[0], arguments[1]);
+            }
+
```

Comment:
I am not convinced that we should add a `Match()` string extension method. I think we should handle this similar to `LIKE`, that just has a `DbFunctions` extension method. So I think this should be removed.

If we decide to include this anyway, this should not be in `MySqlStringMethodTranslator`, but in a new method translator class.